src/neural/cuda/network_cuda.cc

```diff
@@ -490,7 +516,17 @@ class CudaNetwork : public Network {
     }
 
     // Policy head.
-    if (conv_policy_) {
+    if (attn_policy_) {
+      network_[l++]->Eval(batchSize, tensor_mem[0], tensor_mem[2],
+                          tensor_mem[1], scratch_mem, scratch_size_, nullptr,
+                          cublas,
+                          stream);  // Entire Attention policy head except for the policy map
+
+      network_[l++]->Eval(batchSize, (DataType*)opPol, tensor_mem[0], nullptr,
```

Comment:
This calls policy map with direct output to opPol on fp16 - which is not what we do in the other policy output branches.
I replaced this with
      if (fp16) {
        network_[l++]->Eval(batchSize, tensor_mem[1], tensor_mem[0], nullptr,
                            scratch_mem, scratch_size_, nullptr, cublas,
                            stream);  // policy map layer
        copyTypeConverted(opPol, (half*)(tensor_mem[1]),
                          batchSize * kNumOutputPolicy,
                          stream);  // POLICY output
      } else {
        network_[l++]->Eval(batchSize, (DataType*)opPol, tensor_mem[0], nullptr,
                            scratch_mem, scratch_size_, nullptr, cublas,
                            stream);  // policy map layer  // POLICY output
      }
and got 'better' output.